### DISCUSSION TITLE
Consortium - official name problem in SDK 1.11.1 (pdf)

### DISCUSSION THREAD

**Discussion by tompeeszter** [OP-TED/eForms-SDK#1028] [Q&A] [answered]
We noticed that we couldn't see the organisations data of consortium properly under the section 6.1.2 in TED PDF.
Example 1:
- leader of tendering party is OK
-  Official name -> doesn't appear properly. Only one organisation of the consortium appears
![image](https://github.com/user-attachments/assets/72831bc3-3311-47cf-b501-69675a635a13)

Example 2:
- leader of tendering party is OK
-  Official name -> all organisations of the consortium appear aggregated
This case occurs when there ara more than one LOT, and in one of the LOTS there is a consortium but in the other one there is a single tenderer.
![image](https://github.com/user-attachments/assets/2577aa37-efe2-4d35-a663-178900559fd2)

We use SDK 1.11.1. 
Could you please explain this issue?
Which is the correct one?
Thanks in advance!

**Comment by pdonohoe** [comment, marked as answer]
Without access to the XML, I cannot comment on any potential error in the PDF. Please indicate the publication number of the notice, or attach the XML.

However I can explain the intention of the display.
When a tendering party has only one organisation, there is no TenderingPartyLeader, and the single organisation is displayed after the label "Official name"
When a tendering party has more than one organisation, one of them should be identified as the TenderingPartyLeader. The name of this organisation is displayed after the label "Leader of tendering party". The names of the remaining (non-leader) organisations are displayed after the label "Official name".

**Comment by tompeeszter** [comment]
Thank you for the rapid answer!

If I understood well in case of a consortium the leader should not be displayed after the "Official name" label.
I send you an example (both xml and pdf) about the not proper display (consortium leader is also displayed after "Official name" label.)
The example was made on our local test environment.
[TED.txt](https://github.com/user-attachments/files/17166551/TED.txt)
[notice.pdf](https://github.com/user-attachments/files/17166553/notice.pdf)

**Comment by pdonohoe** [comment]
Thank you for the examples. It is indeed a bug in the display. A fix for the Viewer API and the EFX templates will be included in the release of SDK 1.13, and then applied to previous SDK versions. It will take longer to apply the fix to the live TED website.

**Comment by tompeeszter** [comment]
Thanks a lot!
